g2core/report.cpp

```diff
@@ -337,17 +337,18 @@ stat_t sr_status_report_callback()         // called by controller dispatcher
         return (STAT_NOOP);
     }
 
-   // don't send an SR if you the planner is experiencing a time constraint
-   if (!mp_is_phat_city_time()) {
+    // don't send an SR if you the planner is experiencing a time constraint
+    if (!mp_is_phat_city_time()) {
```

Comment:
Accidental?

Reply:
Intentional - those lines were missing 1 space of indentation; figured I'd fix it while in the area.

Reply:
Cool. :smile: